psscriptanalyzer/Manage-Users.ps1

```
@@ -0,0 +1,38 @@
+        [string]$ComputerName,
+        [string]$AdminPassword = "AdminP@ss2024"
+    )
+    $secure = ConvertTo-SecureString -String $AdminPassword -AsPlainText -Force
+    $cred = New-Object System.Management.Automation.PSCredential("Administrator", $secure)
+    Enter-PSSession -ComputerName $ComputerName -Credential $cred
+}
+
+function Reset-UserPassword {
+    param([string]$Username)
+    $newPassword = "Temp_P@ss_$(Get-Random)"
+    $secure = ConvertTo-SecureString -String $newPassword -AsPlainText -Force
```

PSScriptAnalyzer (1.25.0)

[error] File 'Manage-Users.ps1' uses ConvertTo-SecureString with plaintext. This will expose secure information. Encrypted standard strings should be used instead.

(PSAvoidUsingConvertToSecureStringWithPlainText)